flocker/node/agents/test/test_blockdevice.py

```diff
@@ -460,6 +472,10 @@ def _incorrect_device_path_test(self, bad_value):
             volume.blockdevice_id, self.api.compute_instance_id(),
         )
 
+        # XXX There are N different cases to test here.  At least: filesystem
+        # exists with uuid, filesystem exists without uuid, filesystem does not
+        # exist.  Only testing does-not-exist case now.
```

Comment:
:question: Do you normally have XXX comments without assigning them to a person or filing a follow-up task? 

Seems to me that since this is documenting raw work to do, rather than an infelicity in the code, it'd be better to ditch the comment and file a ticket.